pkg/oidc/configure.go

```diff
@@ -93,6 +98,16 @@ func (u *OIDC) Run(ctx context.Context) error {
 		return errors.New("clientID and/or issuerURL are not provided")
 	}
 
+	// Even though in other commands we use comma separated lists, we use a
+	// space separated list here because that's how the OIDC spec defines
+	// scopes. See Section 3.3 of OAuth 2.0
```

Comment:
```
// Even though in other commands we use comma separated lists, we use a
// space separated list here because that's how the OIDC spec defines
// scopes.
```

OIDC spec doesn't force us to have the same format in our CLI, this is our task to convert from one format to another